r1cs/gadgets/std/src/groups/curves/short_weierstrass/short_weierstrass_jacobian.rs

```diff
@@ -1518,9 +1519,13 @@ pub(crate) mod test {
         pub(crate) fn test_incomplete_double_and_add() {
             for _ in 0..10 {
                 let mut cs = ConstraintSystem::<ConstraintF>::new(SynthesisMode::Debug);
+                let mut seed: <XorShiftRng as SeedableRng>::Seed = Default::default();
+                thread_rng().fill(&mut seed);
+                let mut rng = XorShiftRng::from_seed(seed);
+                println!("double and add seed: {:?}", seed);
```

Comment:
You can just use XorShiftRng::from_seed_u64(u64number)

Reply:
Done, thanks for the suggestion :)